packages/admin/src/components/article/AddArticleItemModal.tsx

```
@@ -1,90 +1,139 @@
-} from '../stock/AddInvestItemModal';
+import { color, font } from '@mozu/design-token';
 import { Button, SearchInput } from '@mozu/ui';
-import { color } from '@mozu/design-token';
-import { useState } from 'react';
+import { useState, useEffect } from 'react';
 import { ArticleItem } from './ArticleItem';
 
+// 서버 응답 인터페이스 정의
+interface ArticleItem {
+  id: number;
+  title: string;
```

Biome (1.9.4)

[error] Shouldn't redeclare 'ArticleItem'. Consider to delete it or rename it.

'ArticleItem' is defined here:


(lint/suspicious/noRedeclare)